lib/cocoapods/installer/xcode/pods_project_generator_result.rb

```diff
@@ -0,0 +1,18 @@
+module Pod
+  class Installer
+    class Xcode
+      class PodsProjectGenerator
+        class PodsProjectGeneratorResult
+
+          attr_reader :project
```

Comment:
nit docs are usually added on most classes.

Reply:
Does this look okay? 😬

Reply:
you can add "the project that was generated as part of the generator"

also add the ones in the `def initialize` method. See `pod_target.rb` for example